typescript/cli/src/context/types.ts

```diff
@@ -28,6 +31,8 @@ export interface CommandContext {
   // just for evm chains backward compatibility
   signerAddress?: string;
   strategyPath?: string;
+  multiProtocolProvider?: MultiProtocolProvider;
+  multiProtocolSigner?: MultiProtocolSignerManager;
```

Comment:
Do we need these in readonly commands (warp read etc)? if not can we add these fields as required but in the `WriteCommandContext` type below?

Reply:
made `MultiProtocolProvider ` required in `CommandContext`, but `MultiProtocolSignerManager ` is still optional in `WriteCommandContext` because of the dry run